<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveMesh</class>
 <widget class="QDialog" name="SaveMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Save Mesh...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fileNameLabel">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>File Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="selectDirButton">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="fileNameEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="dataModeLabel">
     <property name="text">
      <string>Data Mode:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="compressionLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Compression:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="dataModeBox">
     <item>
      <property name="text">
       <string>ASCII</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Binary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Appended</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="compressionCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileNameEdit</tabstop>
  <tabstop>selectDirButton</tabstop>
  <tabstop>dataModeBox</tabstop>
  <tabstop>compressionCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveMesh</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveMesh</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
